spec/features/events_spec.rb

```diff
@@ -144,6 +144,16 @@
     end
   end
 
+  scenario "logged in as Organizer I can push the accept all button to accept all applicants" do
+    login(:organizer)
+    @event = FactoryGirl.create :event, :with_diverse_open_applications
+    #@event.max_participants = 3
```

Comment:
it will be later relevant to the user story, but it is not relevant to this pr (since max_participants is not relevant yet)
i changed the comment to be more precise